app/views/conferences/_booths.html.haml

```diff
@@ -0,0 +1,22 @@
+= content_for :splash_nav do
+  %li
+    %a.smoothscroll{ href: '#booths' } Booths
+
+.container
+  .row
+    .col-md-12.text-center
+      %h2 Booths
+  - @conference.booths.where(state: 'confirmed').each_slice(3) do |slice|
+    .row.row-centered
+      - slice.each_with_index do |booth, i|
+        .col-md-4.col-sm-4.col-centered.col-top
+          .thumbnail
+            - if booth.logo_link
+              = link_to(booth.website_url, class: 'thumbnail') do
+                = image_tag booth.picture.large.url
+            .caption
+              %h3.text-center
+                = link_to "#show_descrition_#{i}", "data-toggle"=>"collapse" do
```

Comment:
Perhaps it makes sense to leave the title as it is, without a link and add small link 'Learn more'.